CONTRIBUTING.md

```diff
@@ -20,6 +20,8 @@ Best start by reading the [documentation](https://www.freqtrade.io/) to get a fe
 
 ## Before sending the PR
 
+Do the following if you disabled pre-commit hook when commiting.
```

Comment:
while it tells you what to do without pre-commit - it's not documenting _how_ to set it up.

While `setup.sh` is nice - you can't rely on everyone using the `setup.sh` script - especially developers tend to install things slightly differently depending on their local setup.